#available at /bea/screen_display_mode

#Variables
int8 screen_mode

#default_modes
int8 SLEEP = 0
int8 SURPRISED = 1
int8 NEUTRAL = 2
int8 NERVOUS = 3
int8 HAPPY = 4
int8 VERY_HAPPY = 5
int8 WINK_LEFT = 6
int8 WINK_RIGHT = 7
int8 SAD = 8
int8 SLEEPY = 9